Cars/Service/Employee/IStaffService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Cars.Service
+{
+    interface IStaffService
+    {
```

Comment:
There should be methods, witch will be implemented in the implementationof this interface, I mean EmployeeService. Do same thing here according to the pattern we did.